crates/store/re_grpc_client/src/lib.rs

```diff
@@ -376,3 +387,104 @@ async fn stream_catalog_async(
 
     Ok(())
 }
+
+// Craft a blueprint from relevant chunks and activate it
+fn activate_catalog_blueprint(
+    tx: &re_smart_channel::Sender<LogMsg>,
+) -> Result<(), Box<dyn std::error::Error>> {
+    let blueprint_store_id =
+        StoreId::from_string(StoreKind::Blueprint, "catalog_blueprint".to_owned());
+    let blueprint_store_info = StoreInfo {
+        application_id: ApplicationId::from("redap_catalog"),
+        store_id: blueprint_store_id.clone(),
+        cloned_from: None,
+        is_official_example: false,
```

Comment:
Side note: IIRC this is related to how we deal with analytics. We probably should do something special for these meta-recordings at some point as well.